SpeedandReachFixes/SettingObjects/WeaponStats.cs

```diff
@@ -96,7 +96,7 @@ public float GetSpeed(float current, out bool changed)
         /// <returns>bool</returns>
         public bool ShouldSkip()
         {
-            return Keyword.IsNull || (IsAdditiveModifier && Reach.EqualsWithin(Constants.NullFloat) && Speed.EqualsWithin(Constants.NullFloat));
+            return Keyword.IsNull;// || (IsAdditiveModifier && Reach.EqualsWithin(Constants.NullFloat) && Speed.EqualsWithin(Constants.NullFloat));
```

Comment:
Usually I like just deleting the unused code.  Looking at history is a better way to see how things were, I think.